db/re/item_db.txt

```diff
@@ -10143,12 +10163,19 @@
 19223,Illusion_Cap,Illusion Cap,4,0,,400,,15,,1,0x000654E3,63,2,256,,120,1,14,{ bonus2 bAddClass,Class_All,1; bonus bBaseAtk,(2*getrefine()); },{},{}
 19247,Illusion_Fancy_Flower,Illusion Fancy Flower,4,0,,100,,0,,1,0xFFFFFFFF,63,2,256,,120,0,4,{ bonus bMatkRate,1; bonus bMatk,(2*getrefine()); },{},{}
 //
+19241,C_Magical_Booster_,Costume Magical Booster,4,,,30,,10,,1,0xFFFFFFFF,63,2,2048,0,,,873,{},{},{}
```

Comment:
Wrong weight and itemdescription mention this is a normal mid-helm

19241,Magical_Booster_,Magical Booster,4,,,300,,10,,1,0xFFFFFFFF,63,2,512,0,,,873,{},{},{}